<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetExtraInfos</class>
 <widget class="QWidget" name="WidgetExtraInfos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Llave</string>
       </property>
       <property name="buddy">
        <cstring>txtLlave</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtLlave"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Valor</string>
       </property>
       <property name="buddy">
        <cstring>txtValor</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtValor"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="lwExtraInfos"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btEliminarExtraInfo">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
